service/src/main/java/com/ericsson/eiffel/remrem/generate/controller/RemremGenerateController.java

```diff
@@ -267,7 +267,12 @@ public JsonElement getMessageProtocols() {
         for (MsgService service : msgServices) {
             JsonObject protocolObject = new JsonObject();
             protocolObject.addProperty("name", service.getServiceName());
-            protocolObject.addProperty("edition", service.getProtocolEdition());
+            try {
+                protocolObject.addProperty("edition", service.getProtocolEdition());
+            } catch (NoSuchMethodError | AbstractMethodError e) {
+                // Ignored intentionally in order to ensure compatibility with
```

Comment:
Please, add a warning to log file to clearly indicate that the action failed due to an old used library and suggest upgrade to a fresh one.

Reply:
Sure Roman